dev/NavigationView/NavigationView.cpp

```diff
@@ -1358,6 +1362,36 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    auto newButtonWidths = [this]()
+    {
+        if (DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+        {
+            return static_cast<double>(c_paneToggleButtonWidth);
+        }
+        return CompactPaneLength();
+    }();
+
+    if (const auto backButton = m_backButton.get())
+    {
+        backButton.Width(newButtonWidths);
+    }
+    if (auto&& paneToggleButton = m_paneToggleButton.get())
+    {
+        paneToggleButton.MinWidth(newButtonWidths);
+        if (const auto&& iconGridColumnElement = paneToggleButton.GetTemplateChild(c_paneToggleButtonIconGridColumnName))
+        {
+            if (const auto&& paneToggleButtonIconColumn = iconGridColumnElement.try_as<winrt::ColumnDefinition>())
```

Comment:
>&& [](start = 26, length = 2)

only need && for tracker_refs, in this case just auto is fine for storing the results of the try_as method.

Reply:
Okay, I see. Thanks!